tools/Microsoft.Health.Tools.Sql.Tasks.Tests/Microsoft.Health.Tools.Sql.Tasks.Tests.csproj

```diff
@@ -27,9 +28,12 @@
     <GeneratedFullScriptPath>Schema\$(LatestSchemaVersion).sql</GeneratedFullScriptPath>
   </PropertyGroup>
 
-  <Target Name="PublishSqlTasks" BeforeTargets="CoreCompile">
-    <MSBuild Projects="..\Microsoft.Health.Tools.Sql.Tasks\Microsoft.Health.Tools.Sql.Tasks.csproj" Targets="Publish" Properties="TargetFramework=netstandard2.0" />
+  <Target Name="PublishSqlTasks" BeforeTargets="CoreCompile" Condition=" '$(MSBuildRuntimeType)' == 'Core' ">
```

Comment:
Alternative you could use a [`<choose>`](https://learn.microsoft.com/en-us/visualstudio/msbuild/choose-element-msbuild?view=vs-2022) element.

Reply:
Ok, I will change this element.

Reply:
The choose tag should be the same level like target as this [link ](https://stackoverflow.com/questions/58090751/the-element-when-benerath-element-choose-is-unrecognized)said, so I think it will be better to use condition here.